src/wallet/rpcdump.cpp

```diff
@@ -144,7 +145,7 @@ UniValue importprivkey(const UniValue& params, bool fHelp)
         }
     }
 
-    return NullUniValue;
+    return CBitcoinAddress(vchAddress).ToString();
```

Comment:
I don't see any backwards-compatibility issues with changing a `NullUniValue` to a return value. Any existing user of this RPC would previously have expected (and received) no response, and therefore not be attempting to parse anything specific from the result. The one issue might be if they checked for a lack of a result, as a cheap means of detecting an error (but that would only work for `zcash-cli`, as JSON-RPC has a specific error response mechanism, and only if `zcash-cli` returns errors on stdout, which IDK).